pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -731,6 +731,13 @@ public CompletableFuture<Void> closeAndRemoveReplicationClient(String clusterNam
     public CompletableFuture<Void> closeAsync() {
         try {
             log.info("Shutting down Pulsar Broker service");
+            log.info("Event loops shutting down gracefully...");
+            List<CompletableFuture<?>> shutdownEventLoops = new ArrayList<>();
+            shutdownEventLoops.add(shutdownEventLoopGracefully(acceptorGroup));
+            shutdownEventLoops.add(shutdownEventLoopGracefully(workerGroup));
```

Comment:
Other close tasks rely on EventLoop. E.g. `unloadNamespaceBundlesGracefully`, I don't recommend this change.

Reply:
True. Closing protocol handler first is reasonable because the protocol handler service maybe depends on broker service. How about it ? @Technoboy- 
<img width="959" alt="image" src="https://user-images.githubusercontent.com/4970972/184911740-760e9eb3-563c-4732-9dcd-11eaccc4b996.png">